static/third/marked/lib/marked.js

```diff
@@ -856,6 +865,12 @@ InlineLexer.prototype.userGravatar = function (email) {
   return this.options.avatarHandler(email);
 };
 
+InlineLexer.prototype.timestamp = function (time) {
+  if (typeof this.options.timestampHandler !== 'function')
+    return '!time(' + time + ')';
```

Comment:
Stylistically, we generally always use `{}` on if statements; this file is excluded from the linter because it's mostly third-party code, but we should probably fix this here since it's our own additions to third-party code.